Source/Core/DolphinQt2/Settings/InterfacePane.cpp

```diff
@@ -165,6 +192,8 @@ void InterfacePane::ConnectLayout()
   connect(m_checkbox_pause_on_focus_lost, &QCheckBox::clicked, this, &InterfacePane::OnSaveConfig);
   connect(m_checkbox_hide_mouse, &QCheckBox::clicked, &Settings::Instance(),
           &Settings::SetHideCursor);
+  connect(m_checkbox_use_userstyle, &QCheckBox::clicked, this,
```

Comment:
The signal that should be used is `stateChanged`.

https://doc.qt.io/qt-5/qcheckbox.html

>Whenever a checkbox is checked or cleared, it emits the signal stateChanged(). Connect to this signal if you want to trigger an action each time the checkbox changes state. You can use isChecked() to query whether or not a checkbox is checked.